rllib/algorithms/ppo/tf/ppo_tf_rl_module.py

```diff
@@ -20,32 +20,13 @@ def __init__(self, *args, **kwargs):
         TfRLModule.__init__(self, *args, **kwargs)
         PPORLModuleBase.__init__(self, *args, **kwargs)
 
-    # TODO(Artur): Comment in as soon as we support RNNs from Polciy side
-    # @override(RLModule)
-    # def get_initial_state(self) -> NestedDict:
-    #     if hasattr(self.encoder, "get_initial_state"):
-    #         return self.encoder.get_initial_state()
-    #     else:
-    #         return NestedDict({})
-
     @override(RLModule)
     def _forward_inference(self, batch: NestedDict) -> Mapping[str, Any]:
         output = {}
 
-        # TODO (Artur): Remove this once Policy supports RNN
-        batch = batch.copy()
```

Comment:
@sven1977 you recently added this copy. Did you do that because the bad should not be modified when using eager tracing or something like that?

Reply:
Correct, it breaks stuff. Apparently, you cannot alter the incoming data dict in any way (when tracing).